yarn.lock

```diff
@@ -2334,13 +2340,30 @@
     "@endo/lockdown" "^0.1.28"
     "@endo/promise-kit" "^0.2.56"
 
+"@endo/init@^1.0.1":
+  version "1.0.1"
+  resolved "https://registry.yarnpkg.com/@endo/init/-/init-1.0.1.tgz#e83d52f384089cdb3119d47cd778fe0d80e92a97"
+  integrity sha512-9JPCwiQU6wc5SC2jc10VLOV95FxqBUU+94SR2hghPDrJQQdQ6+Kic8d2DFzEH8vSIGpR9ibBTnAzFjmPfVuAxA==
+  dependencies:
+    "@endo/base64" "^1.0.0"
+    "@endo/eventual-send" "^1.0.1"
+    "@endo/lockdown" "^1.0.1"
+    "@endo/promise-kit" "^1.0.1"
+
 "@endo/lockdown@0.1.28", "@endo/lockdown@^0.1.28":
   version "0.1.28"
   resolved "https://registry.yarnpkg.com/@endo/lockdown/-/lockdown-0.1.28.tgz#43f23dcbb12b6ebd3ad2a3dc8c6bb3609dd9e95f"
   integrity sha512-YqurtDU23+0kuWq4J2c94HyRB1aqSB8xEwrx5xTZA9IY/anrtppEiTFGU8tQXqZFhE6bfRzSGWDIVKaXCcm4Lw==
   dependencies:
     ses "^0.18.4"
 
+"@endo/lockdown@^1.0.1":
```

Comment:
this dupes the other endo/lockdown. consider running `npx yarn-deduplicate` and/or bumping the other Endo deps in this repo to their latest release

Reply:
I rolled back to 0.5.60 in an attempt to be consistent with existing deps; I'll leave bumping the deps to others.

I did de-dup.